content/articles/creating-movies-and-animations-using-matlab/index.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /creating-movies-and-animations-using-matlab/
+title: Creating movies and animations using Matlab
+description: In this article, we will look at how you can create an animation using Matlab. We will also look at the various steps involved and use the Matlab inbuilt functions to make the activity simpler.
+author: florence-atieno
+date: 2021-10-09T00:00:00-11:30
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/creating-movies-and-animations-using-matlab/hero.jpg
+    alt: Creating movies and animations using Matlab Hero image
+---
+
+Animation is a series of still images one after another. If you show these images together in rapid succession, the brain interprets them as continuous fluid motion.
+<!--more-->
+The animation follows a similar workflow to that of creating a flip-flop book. A flip-flop book is a booklet with a series of images that gradually change from one page to the next. 
+
+When you view the pages quickly, the images appear to animate by simulating motion or some other change. Animation has a wide advantage and is widely used in the science and engineering field. 
```

Comment:
A lot of your paragraphs are quite small and spaced out. Would be good to bunch them up a bit so your article flows better. Especially your single line paragraphs. This is prevalent throughout your article.